doc/source/parameters/cfd/simulation_control.rst

```diff
@@ -1,108 +1,177 @@
+    #---------------------------------------------------
     # Log frequency
-    set log frequency                = 1
+    set log frequency     = 1
   
     # Display precision when writing to log
-    set log precision                = 6
-  
-    # Maximum CFL value
-    set max cfl                      = 1
-  
-    # The kind of solver for the linear system. Choices are
-    # <steady|steady_bdf|bdf1|bdf2|bdf3|sdirk2|sdirk3>.
-    set method                       = steady
-  
-    # Number of mesh adaptation (for steady simulations)
-    set number mesh adapt            = 0
-  
-    # Output the boundaries of the domain along with their ID
-    set output boundaries            = false
-  
-    # The control for the output of the simulation results. Results can be either
-    # outputted at constant iteration frequency or at constant time
-    set output control               = iteration
-  
-    # Output frequency
-    set output frequency             = 1
-  
+    set log precision     = 6
+
+    #---------------------------------------------------
+    # Output file parameters
+    #---------------------------------------------------
+    # File output path
+    set output path       = ./
+
     # File output prefix
-    set output name                  = out
+    set output name       = out
+
+    # The control for the output of the simulation results
+    set output control    = iteration
   
-    # File output path
-    set output path                  = ./
+    # Output frequency
+    set output frequency  = 1
   
     # Output time
-    set output time                  = 1
-  
-    # Scaling factor used in the iterations necessary to start-up the BDF
-    # schemes.
-    set startup time scaling         = 0.4
-  
-    # Tolerance at which the simulation is stopped
-    set stop tolerance               = 1e-10
-  
+    set output time       = 1
+
+    # Maximal number of vtu output files
+    set group files       = 1
+
+    # Output the boundaries of the domain along with their ID
+    set output boundaries = false
+    
     # Subdivision of mesh cell in postprocessing
-    set subdivision                  = 1
-  
-    # End time value of the simulation
-    set time end                     = 1
-  
-    # Time step value
-    set time step                    = 1.
+    set subdivision       = 1
+
   end
 
-* The ``group files`` parameter controls the number of vtu files generated in a parallel simulation. This parameter allows to reduce the number of files generated when the simulation is run with a large number of processors. Setting group files to 1 ensures that there is a single vtu file generated. In this case, the file is written using MPI IO functionalities. The value for this parameter should always be a compromise between keeping a low number of files but preventing excessive MPI communications. We have found that the default value of 1 does not have a significant impact on performance on Compute Canada clusters.
```

Comment:
I prefer the previous version. This is a very subtly parameter and I rather have it more verbose than not verbose

Reply:
Don't worry, I kept all this information, this is of course relevant! I just put it in the tip-box just below, I think it is clearer when displayed in a web browser